config/sync/core.base_field_override.node.cs_spotlight.promote.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 44a46596-d758-41ed-a868-f5af2dc9b9e1
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.cs_spotlight
+id: node.cs_spotlight.promote
```

Comment:
I thought we decided to not name items based on clients. 
https://docs.google.com/spreadsheets/d/1ATP3boyFio_s2BYjvzBoOt29p7aQrQV3HB-kZOZwp0g/edit#gid=0

Everything should still be prefixed with su_ and stanford_ as we want collisions and easy poachable code.

Reply:
i thought that was for jackalope since it was planned to make it universal. these are client specific

Reply:
Yeah, we decided as a group we want to have collisions and no matter if it is client-specific or not we follow the same naming conventions.

Reply:
thats ringing a bell now

Reply:
Sorry, I understand this is a non-trival change to this PR.

Reply:
because this is a forked repo, those collisions are gonna get bad

Reply:
its not so bad. find and replace `cs_` to `su_` and run `rename 's/cs_/su_/' *.yml` a couple times :smile: